solidity/test/bridge/Bridge.Frauds.test.ts

```diff
@@ -1402,6 +1416,20 @@ describe("Bridge - Fraud", () => {
                     ).to.have.been.calledOnceWith(walletDraft.ecdsaWalletID)
                   })
 
+                  it("should call the ECDSA wallet registry's seize function", async () => {
+                    expect(walletRegistry.seize).to.have.been.calledOnceWith(
+                      fraudSlashingAmount,
+                      fraudNotifierRewardMultiplier,
+                      await thirdParty.getAddress(),
+                      ecdsaWalletTestData.walletID,
+                      walletMembersIDs
+                    )
+                  })
+
+                  // TODO: Check if the gas consumption of functions calling `seize`
+                  //       is not too high (use a real `staking` and `walletRegistry`).
+                  //       Perhaps add a separate deployment with the non-mocked contracts
+                  //       or test it in a system test?
```

Comment:
This will have to be a separate file with a real `TokenStaking` and real `WalletRegistry` deployment plus the entire setup. We should assert the gas price in the worst case is sane. Ideally, we could use the mechanism @nkuba introduced in https://github.com/keep-network/keep-core/pull/2926. See `solidity/ecdsa/test/helpers/gas.ts` and how `assertGasUsed` is used.

Reply:
@pdyraga I'm working on it.